### DISCUSSION TITLE
How do I prevent the "yarn start" process from stopping?

### DISCUSSION THREAD

**Discussion by MichaelBelgium** [umami-software/umami#2268] [Q&A] [answered]
I self-host umami and I just discovered it has missed 5 days of analytics because the yarn process stopped somehow - which results in umami being offline.

I always execute  `yarn start &` to execute it and send it in the background. Is there a better way perhaps?

**Comment by FHU-yezi** [comment]
pm2 maybe help.

**Reply by MichaelBelgium** [reply]
Which js file do i need to pass then to `pm2 start`

**Reply by MichaelBelgium** [reply]
Ah looks like[ digital ocean docs](https://umami.is/docs/running-on-digitalocean) use pm2 also. 

```
pm2 start npm --name umami -- start
pm2 save
```

Unfortunately fails for me `[PM2][ERROR] File ecosystem.config.js not found`

**Reply by MichaelBelgium** [reply, marked as answer]
Found it

The command should be [`pm2 start npm --name "next" -- start`](https://github.com/umami-software/umami/issues/16#issuecomment-675893795)

Quotes were required i guess
